frontend/packages/console-shared/src/components/multi-tab-list/MultiTabListPage.tsx

```diff
@@ -19,6 +20,7 @@ interface MultiTabListPageProps {
   pages: Page[];
   match: Rmatch<any>;
   secondaryButtonAction?: SecondaryButtonAction;
+  navMenu?: string;
```

Comment:
The navigation and a page are not connected, or not connected in this direction. This props sounds to me, oh I can add a (sub) navigation configuration the MultiTabList, but that's not the case here.

Recommended names: `titleKey` or `telemetryPrefix`

Both makes it more clear that this will reflect the (displayed) page title more then the navigation title:

![image](https://user-images.githubusercontent.com/139310/229844213-82c806f8-1dea-442c-9ba6-a70af7725874.png)

![image](https://user-images.githubusercontent.com/139310/229844302-cc286d7f-151a-43fc-b2f1-471316bee9e3.png)

Reply:
Changed the name to `telemetryPrefix`